README.md

```diff
@@ -54,46 +54,66 @@ We recommend that you link to `<url for repository's GitHub pages>/workshop/HOME
 
-1. Make sure you have [Bundler](https://bundler.io/) installed. Installation can be accomplished with `gem install bundler`.
- 
-  If you encounter a permissions error, it was solved (using this StackOverflow post as a guide) with the following on Mojave with [Homebrew](https://brew.sh/):
+### Installing GitHub pages dependencies locally
 
-  ```
-  # install rbenv
-  brew update
-  brew install rbenv
+Installing the dependencies for GitHub pages is best done in a separate ruby environment, managed by [rbenv](https://github.com/rbenv/) and [Bundler](https://bundler.io)
 
-  # add to ~/.bash_profile
-  echo 'eval "$(rbenv init -)"' >> ~/.bash_profile
-  ```
+The following instructions are for installation on Mac OS.
+1. [Install rbenv](https://github.com/rbenv/rbenv#installation).
+The easiest, and recommended, installation is through Homebrew:
+```
+# install rbenv
+brew update
+brew install rbenv ruby-build
+```
+Alternatively, you can install with [rbenv-installer](https://github.com/rbenv/rbenv-installer#rbenv-installer) which will work even without Homebrew (though it uses Homebrew if you have it).
 
-  Start a new shell to initialize `rbenv`.
+2. Install rbenv for your shell for your shell.
+Since this will depend on which shell you are using, you will want to run `rbenv init` to find the correct command to add you your shell configuration file.
+Note that this command does _not_ actually perform the initialization: it only prints instructions for what line to add to which file!
+Be sure to add that line to the _end_ of the configuration file indicated.
 
+4. Once you have modifided your shell configuration, start a new shell to initialize `rbenv`.
+
+5. Install a current stable ruby version
+  ```
+  # install and set up v3.1.3 of Ruby
+  rbenv install 3.1.3
   ```
-  # install and setup v2.7.1 of Ruby
-  rbenv install 2.7.1
-  rbenv shell 2.7.1
-  rbenv rehash
-  ``` 
 
-  Now you're ready to run `gem install bundler`. 
+6. Activate your ruby version. Here you have a few options:
 
-2. Install Jekyll (`gem install jekyll`).
-3. Run `bundle install` from the root of this repository, where the `Gemfile` is located.
-4. Run `bundle exec jekyll serve` and navigate to the server address.
+   a. If you want to set the global ruby version that will be used wherever you use `ruby`, you can use
+   ```
+   rbenv global 3.1.3
+   ```
 
-If you've already followed the steps above and are in a new shell, run the following steps to build locally:
+   b. To only use this version of ruby within the repository only, navigate to the root of this repository and use:
+   ```
+   rbenv local 3.1.3
+   ```
+   This will create a `.ruby-version` file that will automatically activate this version of ruby whenever you are in this directory.
 
+7. Now you should be ready to install Bundler:
 ```
-rbenv shell 2.7.1
-rbenv rehash
-bundle exec jekyll serve  
+gem install bundler
```

Comment:
Just noting that at first I was getting an error here: 
```
you don't have write permissions for the /library/ruby/gems/2.6.0 directory
```
I restarted my terminal and then ran the command again and no error, so maybe we need to add a line about starting a new shell? 
But other than that everything worked for me.

Reply:
Did you start a new shell at step 4 (after modifying your `.bashrc` or `.zshrc`?

Reply:
Yes I did it after that step and then again when I saw that error.

Reply:
So interesting! I would not expect it to require two restarts, but I guess maybe it does.

Reply:
Did you do a global or local activation? I wonder if it doesn't rehash for a global until restart...

Reply:
I did local first and then when it didn't work I tried global and still got the same error.

Reply:
I just did a local activation. A restart was not required.

Reply:
I also received the same error as Ally here after doing a global activation -- which was also fixed by restarting the terminal.